buidl/hash.py

```diff
@@ -0,0 +1,48 @@
+try:
+    from buidl.cecc import tagged_hash
+except ModuleNotFoundError:
+    import hashlib
+
+    def tagged_hash(tag: bytes, msg: bytes) -> bytes:
+        tag_hash = hashlib.sha256(tag).digest()
+        return hashlib.sha256(tag_hash + tag_hash + msg).digest()
+
+
+def hash_aux(msg):
```

Comment:
All of the `tagged_hash`es feels wrong (vs a DRY enum-style construction), but is very readable and using nice helper methods makes it easier for the caller downstream.

Just commentary, probably makes most sense as-is. I get that it was previously this way in `helper.py`, so not really a comment on this PR.